core/src/main/java/nu/yona/server/subscriptions/service/UserPrivateDataBaseDto.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License, v.
+ * 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server.subscriptions.service;
+
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import nu.yona.server.device.service.DeviceBaseDto;
+import nu.yona.server.goals.service.GoalDto;
+
+public class UserPrivateDataBaseDto
```

Comment:
This has no JSON annotation anymore. Are the `@JsonIgnore` properties in this class needed?

Reply:
Yes, as the subtypes do JSON serialization. That includes the inherited properties.